README.md

```diff
@@ -67,6 +67,7 @@ To compile and run the Zilliqa codebase, you will need the following dependencie
 * `libevent`
 * `miniupnpc`
 * `ocl-icd-opencl-dev`
+* `cuda9.2`
```

Comment:
Let's only put the minimal dependencies here. I can help to update the README if you want.

Reply:
Thanks, i will handle it.